src/features/linter-provider.ts

```diff
@@ -107,7 +107,7 @@ export default class FortranLintingProvider {
     );
     let argList = [
       ...args,
-      getIncludeParams(includePaths), // include paths
+      ...(getIncludeParams(includePaths)), // include paths
```

Comment:
```suggestion
      ...getIncludeParams(includePaths), // include paths
```
No need for the extra parentesis.

Reply:
removed.